src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -43,7 +49,7 @@ void buildSearchConceptsQuery(CloudPlatform platform) {
               "SELECT c.concept_name, c.concept_id, COUNT(DISTINCT o.person_id) AS count "
                   + "FROM concept AS c "
                   + "JOIN observation AS o ON o.observation_concept_id = c.concept_id "
-                  + "WHERE (c.domain_id = 'observation' "
+                  + "WHERE (c.domain_id = 'Observation' "
```

Comment:
This kind of change is fine but I don't want us to feel like we need to update our test data when the real data changes, unless there's a specific case we need to test for. For that reason I often make test data look nothing like the real data so there's no confusion about what needs to be changed when a real data change occurs.